x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/AnalyzerTestUtils.java

```diff
@@ -111,7 +111,7 @@ public static IndexResolution loadMapping(String resource, String indexName, Ind
     }
 
     public static IndexResolution loadMapping(String resource, String indexName) {
-        EsIndex test = new EsIndex(indexName, EsqlTestUtils.loadMapping(resource));
+        EsIndex test = new EsIndex(indexName, EsqlTestUtils.loadMapping(resource), Map.of(indexName, IndexMode.STANDARD));
```

Comment:
Added this to try to make it as realistic as possible (All indices with concrete "counterparts"